tests/wrapped_functions.cpp

```diff
@@ -0,0 +1,21 @@
+#define BOOST_TEST_MODULE dcgp_expression_ann_test
+#include <boost/test/unit_test.hpp>
+#include <cmath>
+#include <stdexcept>
+
+#include <dcgp/kernel.hpp>
+#include <dcgp/kernel_set.hpp>
+using namespace dcgp;
+
+BOOST_AUTO_TEST_CASE(pdiv)
+{
+    // Initialize kernel
+    kernel_set<double> pdiv_set({"pdiv"});
```

Comment:
if here you check that this does not throw use BOOST_NO_THROW macro. Otherwise its just a line that is unused

Reply:
That line is used to get the kernel. For easy of use I extract the `pdiv` kernel from the set (which is the next line).